packages/types/src/coreExtensions/contextMenu.ts

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Any particular reason or did you just forget to remove it?

Reply:
The prettier formatting for the unions here was pretty bad, I've changed the directives to only affect that part of the file.